skbio/diversity/beta/_base.py

```diff
@@ -59,18 +55,38 @@ def pw_distances(metric, counts, ids=None, **kwargs):
     scipy.spatial.distance.pdist
     pw_distances_from_table
 
+    Notes
+    -----
+    The value that you provide for for ``metric`` can be either a string (e.g.,
+    ``"unweighted_unifrac"``) or a function
+    (e.g., ``skbio.diversity.beta.unweighted_unifrac``). The metric should
+    generally be passed as a string, as this often uses an optimized version
+    of the metric. For example, passing  ``"unweighted_unifrac"`` (a string)
+    will be hundreds of times faster than passing the function
+    ``skbio.diversity.beta.unweighted_unifrac``. The latter is faster if
+    computing only one or a few distances, but in these cases the difference in
+    runtime is negligible, so it's safer to just err on the side of passing
+    ``metric`` as a string.
+
     """
-    _skbio_metrics = _get_skbio_metrics()
     num_samples = len(counts)
     if ids is not None and num_samples != len(ids):
         raise ValueError(
             "Number of rows in counts must be equal to number of provided "
             "ids.")
-    if metric in _skbio_metrics:
-        metric = _skbio_metrics[metric]
 
-    if callable(metric):
+    if metric == 'unweighted_unifrac':
```

Comment:
If the available metrics for alpha diversity are discoverable (see my comment above), we should put the same structure in place for the beta diversity metrics.